origins_ckeditor_enhancements/origins_ckeditor_enhancements.module

```diff
@@ -15,15 +15,19 @@ use Drupal\Core\Form\FormStateInterface;
  * @param array $types
  *   Text format process.
  */
-function remove_format_help_element_info_alter(array &$types) {
+function origins_ckeditor_enhancements_element_info_alter(array &$types) {
   $types['text_format']['#process'][] = 'remove_format_help_process_text_format_element';
 }
 
 /**
  * Text input element process handler that will remove filter-help, 'About text formats'.
  */
 function remove_format_help_process_text_format_element(&$element, FormStateInterface $form_state, &$complete_form) {
-  $element['format']['guidelines']['#access'] = FALSE;
-  $element['format']['help']['#access'] = FALSE;
+  // Only remove the filter help if the user doesn't have permission
+  // to use the 'Full HTML' text format.
```

Comment:
Can I just check - is this because everyone can use the Basic HTML profile?

Reply:
Yup, and if you only have access to the one profile there is no point in displaying the profile link.

Reply:
Yes, exactly that